### PR TITLE
refactor: make withdraw compatible with the light client

### PR DESCRIPTION
**Pull request by homura**
Fixes: https://github.com/Magickbase/neuron-public-issues/issues/316

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1731,6 +1745,13 @@
   dependencies:
     jsbi "^4.1.0"
 
+"@ckb-lumos/bi@0.21.0-next.2":
```
Comment (Keith-CY) [inline yarn.lock:1748]: Multiple versions of lumos are introduced

1. @ckb-lumos/bi@0.20 at line 1735
2. @ckb-lumos/bi@0.21.0-next.1 at line 1742
3. @ckb-lumos/bi@0.21.0-next.2 at line 1748

Comment (homura) [reply yarn.lock]: Lumos's version is unified and fixed into the same one

**Review by Keith-CY — APPROVED** [review]

**Review by yanguoyu — APPROVED** [review]

**Comment by Keith-CY** [comment]
Please have a review @silySuper @FrederLu

**Review by silySuper — APPROVED** [review]

**Pull request merged**